pages/stats.js

```diff
@@ -82,15 +83,21 @@ const SortableHeader = ({
   );
 };
 
-const StatsPage = ({ stats }) => {
+const StatsPage = ({ stats, recentStats }) => {
+  const [order, setOrder] = useState('desc');
+  const [selectedSortKey, setSelectedSortKey] = useState('avgScore');
+  const [statsRange, setStatsRange] = useState('recent');
+
+  let relevantStats;
+  if (statsRange === 'recent') relevantStats = recentStats;
+  if (statsRange === 'allTime') relevantStats = stats;
+
   const {
     globalScoresByHole,
     topRoundsByScore,
     statsByPlayer,
     MIN_ROUNDS,
-  } = stats;
-  const [order, setOrder] = useState('desc');
-  const [selectedSortKey, setSelectedSortKey] = useState('avgScore');
```

Comment:
I'm confused - why are you deleting these? They're still being used, right?

Reply:
They are just being by moved a few lines up.